pkg/sqlutil/log.go

```diff
@@ -0,0 +1,227 @@
+package sqlutil
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/jmoiron/sqlx"
+	"github.com/lib/pq"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+)
+
+var promSQLQueryTime = promauto.NewHistogram(prometheus.HistogramOpts{
+	Name:    "sql_query_timeout_percent",
+	Help:    "SQL query time as a percentage of timeout.",
+	Buckets: []float64{10, 20, 30, 40, 50, 60, 70, 80, 90, 100, 110, 120},
```

Comment:
For my understanding: why stop/end at 120?

Reply:
The response to context cancellation is generally very fast, so we don't see values beyond this point very often :shrug: 
Or at least we didn't when I calibrated this scale for the original one that I copied here. Could double check in the grafana stats.